### DISCUSSION TITLE
How to check which account I'm logged into github copilot with?

### DISCUSSION THREAD

**Discussion by Constantine-Ketskalo** [community/community#107437] [New to GitHub] [answered]
I've logged into the github copilot already. But I want to make sure that I'm logged in with a correct account. It doesn't show anywhere which account I'm logged in with. And I think that I've logged in last time with vs code instead of a browser. But I don't see in my vs code which account I'm logged in with neither.
So how do I check which account I'm logged in with in both vs code and github copilot?

Search over internet and FAQs didn't show me anything about that.

**Comment by zvi-quantivly** [comment]
Hi @Constantine-Ketskalo, I just ran into the same issue. You should be able to see the username of the logged-in user in the welcome message displayed when you open the chat view (_Command Palette -> Chat: Focus on GitHub Copilot View_). To change the user, I had to remove and reinstall the Copilot extension, open the chat view, and sign in with the correct account. HTH.

**Reply by Wilson-MedAR** [reply]
According to GitHub copilot (because I had the same issue so I asked it):
To change the user you're logged in as with GitHub Copilot in VS Code, follow these steps:

1. Open VS Code.
2. Go to the Command Palette by pressing Ctrl+Shift+P (Windows/Linux) or Cmd+Shift+P (Mac).
3. Type GitHub: Sign Out and select it.
4. After signing out, open the Command Palette again.
5. Type GitHub: Sign In and select it.
6. Follow the prompts to sign in with your desired GitHub account.

This will change the user for GitHub Copilot in VS Code.

**Comment by kaweendras** [comment]
If you have copilot chat enabled, you can see the username of the account when you ask copilot something in the chat
![Untitled](https://github.com/community/community/assets/22750622/d3fce965-e4ed-4820-ae73-6c173c277e25)

**Reply by Constantine-Ketskalo** [reply]
Seems like github copilot and github copilot chat work differently. More details [here](https://github.com/orgs/community/discussions/113587).
Github copilot chat has more info about account I've logged in with, but it's inconvenient to use. Github copilot itself is opposite: convenient to use, but not clear which account am I logged in with.

**Comment by abhishekjaiswar221** [comment, marked as answer]
### @Constantine-Ketskalo You can see in the Accounts section in the lower left corner of the VS Code, which looks like this:

![Screenshot 2024-06-21 173949](https://github.com/community/community/assets/113609441/d56bda8a-f2ed-4248-a90e-936afdbdf0f0)

**Reply by jdavidferreira** [reply]
> ### @Constantine-Ketskalo You can see in the Accounts section in the lower left corner of the VS Code, which looks like this:
> ![Screenshot 2024-06-21 173949](https://private-user-images.githubusercontent.com/113609441/346209259-d56bda8a-f2ed-4248-a90e-936afdbdf0f0.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTQ5NjAwODcsIm5iZiI6MTc1NDk1OTc4NywicGF0aCI6Ii8xMTM2MDk0NDEvMzQ2MjA5MjU5LWQ1NmJkYThhLWYyZWQtNDI0OC1hOTBlLTkzNmFmZGJkZjBmMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwODEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDgxMlQwMDQ5NDdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01YTRjOGYxOWJjMDhjYzdmNTBkNDc3ZmQ0OTM1MDg4NWJhYTM0ZGQxOGQ0ZGNiNTE0YmM0ODA0NTg0YzkwZjdlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.GjDzoTPvy-lkNH4zeQzJOa0WKVRzjsijKf6kkirXsmo)

Mine looks like this:
<img width="318" height="268" alt="image" src="https://github.com/user-attachments/assets/eb94ea02-1e2a-47a1-855e-7f340846cbc8" />
I have Github Copilot on both accounts.

Vscode v1.103.0

**Comment by KLtechworks** [comment]
I followed the guide by Abhishekjaiswar221: just click on the account, sign out, and then sign in to the account you want to log in to again. It works!

**Comment by megafluffles** [comment]
You can switch accounts on the fly, not sure if it's a recent addition.

Go to Extensions, Click the Settings Icon on the Github Copilot extension, select "Account preferences", then you select which account to use:

<img width="633" height="166" alt="image" src="https://github.com/user-attachments/assets/9bd02664-478d-4d69-a361-5c27354501cf" />

**Comment by Beloded** [comment]
You can find required button in plugins section:

<img width="919" height="534" alt="image" src="https://github.com/user-attachments/assets/497e684f-1a56-4d0c-ae1c-d67dfe43e3c2" />
